### PR TITLE
[1953] Switch to Course.IsSend attribute

### PR DESCRIPTION
**Pull request by timabell**
To match database change made in rails, so that publish to find
continues to use accurate data.

[manage-courses-backend #704](https://github.com/DFE-Digital/manage-courses-backend/pull/704/files)

### Context

Move fast and break things (in rails)

### Changes proposed in this pull request

Update to match new db schema on the rails side - [manage-courses-backend #704](https://github.com/DFE-Digital/manage-courses-backend/pull/704/files)

### Guidance to review

* Test against database generated by Rails

### CODE REVIEW

File: tests/ManageCourses.Tests/UnitTesting/CourseMapperTests.cs [hunk 1]
```diff
@@ -35,10 +35,10 @@ public void MapToSearchAndCompareCourse_ProviderLocation()
         }
 
         [Test]
-        public void MapToSearchAndCompareCourse_IsSen()
+        public void MapToSearchAndCompareCourse_IsSend()
         {
             var ucasCourse = GenerateUcasCourse();
-            ucasCourse.CourseSubjects = new Collection<CourseSubject>{new CourseSubject{ Subject = new Subject{SubjectCode = "u3", SubjectName ="special educational needs"}}};
```
Comment (richrace) [inline tests/ManageCourses.Tests/UnitTesting/CourseMapperTests.cs]: Is this sending Data to UCAS?

Comment (timabell) [reply tests/ManageCourses.Tests/UnitTesting/CourseMapperTests.cs]: No, it used to be from UCAS

**Comment by timabell** [comment]
[Update c# to use new SEND attribute](https://trello.com/c/DnoNVabp/1953-update-c-to-use-new-send-attribute)

**Comment by timabell** [comment]
This can be merged once https://github.com/DFE-Digital/manage-courses-backend/pull/704is merged.

It can be shipped once https://github.com/DFE-Digital/manage-courses-backend/pull/704 is shippeed

**Pull request merged**